azure-toolkit-libs/azure-toolkit-springcloud-lib/src/main/java/com/microsoft/azure/toolkit/lib/springcloud/service/SpringCloudDeploymentService.java

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.toolkit.lib.springcloud.service;
+
+import com.microsoft.azure.management.appplatform.v2020_07_01.implementation.AppPlatformManager;
+import com.microsoft.azure.management.appplatform.v2020_07_01.implementation.DeploymentResourceInner;
+import com.microsoft.azure.toolkit.lib.springcloud.model.SpringCloudAppEntity;
+import com.microsoft.azure.toolkit.lib.springcloud.model.SpringCloudDeploymentEntity;
+
+public class SpringCloudDeploymentService {
+
+    private final AppPlatformManager client;
+
+    public SpringCloudDeploymentService(AppPlatformManager client) {
+        this.client = client;
+    }
+
+    public void start(final SpringCloudDeploymentEntity deployment) {
+        this.start(deployment.getName(), deployment.getApp());
+    }
+
+    public void stop(final SpringCloudDeploymentEntity deployment) {
+        this.stop(deployment.getName(), deployment.getApp());
+    }
+
+    public void restart(final SpringCloudDeploymentEntity deployment) {
+        this.restart(deployment.getName(), deployment.getApp());
+    }
+
+    public void remove(final SpringCloudDeploymentEntity deployment) {
+        this.remove(deployment.getName(), deployment.getApp());
+    }
+
+    public void start(String deployment, final SpringCloudAppEntity app) {
```

Comment:
why it is public? if it need to be public, SpringCloudAppEntity should be taken as the first argument.